samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
yeah so this 3rd state which is opt out of default, but continue is pretty hard to name and also pretty confusing to explain.